content/_code-samples/escrow/py/account_escrows.py

```diff
@@ -0,0 +1,53 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountObjects
+from xrpl.utils import drops_to_xrp, ripple_time_to_datetime
+
+# Retreive all escrows created or received by an account, formatted
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to the testnetwork
+
+account_address = "r9CEVt4Cmcjt68ME6GKyhf2DyEGo2rG8AW"
```

Comment:
I'd prefer either the code take user input for an address or use a real pre-filled address that has escrows on the Testnet.

Reply:
It does "r9CEVt4Cmcjt68ME6GKyhf2DyEGo2rG8AW"